backend/addcorpus/validation/creation.py

```diff
@@ -122,6 +122,13 @@ def validate_name_is_not_a_route_parameter(value):
             f'{value} cannot be used as a field name, because it is also a route parameter'
         )
 
+
+def validate_name_has_no_ner_suffix(value):
+    if value.endswith('_ner'):
+        raise ValidationError(
+            f'{value} cannot be used as a field name: the suffix `_ner` is reserved for annotated_text fields'
+        )
```

Comment:
Neat!

Since field names must be slugs, would it be possible to use a suffix that doesn't overlap? (e.g. `:ner` instead of `_ner`)

Reply:
`SlugField` has a built-in validator that only allows numbers, dashes and hyphens, doesn't it? I didn't think the `validators` argument when initializing a `models.SlugField` overrides this basic validator, but is added to it.

Reply:
Nevermind, the validators do indeed replace it. When starting on the project, I remember using a colon as a separator, and running into the `SlugField` in-built validator. It's very likely that back then we didn't have the validator override in place yet.

Reply:
Oh, good to know!

I actually meant: as I understand, this validation is meant to prevent a situation where a corpus has two fields like `content_translated` and `content_translated_ner`, where the latter is just a normal content field. This is a problem because the named entity field for `content_translated` would also be called `content_translated_ner`.

You could keep the slug validation as-is, and then implement "special" suffixes with non-slug characters. So `content_translated` would have a named entity field `content_translated:ner`. So there is never any possible overlap between the slug name and any reserved suffixes.